src/KafkaW/BrokerSettings.h

```diff
@@ -1,17 +1,15 @@
 #pragma once
 
+#include <librdkafka/rdkafka.h>
```

Comment:
This seems like a regression to me, better to forward declare than include in the header?

Reply:
Possibly yes. Why though?

Reply:
I do not think that is a very good argument for a simple (quick to compile project) such as this one but I am not going to press the issue. I will make changes to revert to forwarding.